extensions/menubar-weather/package.json

```diff
@@ -107,6 +107,24 @@
         }
       ]
     },
+    {
+      "name": "tempType",
+      "type": "dropdown",
+      "title": "Display in menubar",
+      "description": "Forecast of feels-like",
+      "required": true,
```

Comment:
Hi! Thank you so much for your contribution to Raycast!
My name is Milena and I will be reviewing your update today :)

Adding `required` preferences might break your extension in case an older user doesn't have it set. Do you think it would be ok to remove the `required` as you already have a default value and the user input is controlled by a dropdown of options? And, of course, if the user has not selected anything, always fallback to the default value.

Reply:
Not really. This is a known issue right now.